lib/pymedphys/docs/users/howto/gamma/from-dicom.ipynb

```diff
@@ -4,9 +4,9 @@
    "cell_type": "markdown",
```

Comment:
<p>So is the main theme of this PR to show explicit support for different orientations?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymedphys/pymedphys/pull/1770/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
yes, it is.